base_synchro/wizard/base_synchro.py

```diff
@@ -174,7 +178,8 @@ def get_id(self, object_id, id, action):
         line_pool = self.env["base.synchro.obj.line"]
         field_src = (action == "u") and "local_id" or "remote_id"
         field_dest = (action == "d") and "local_id" or "remote_id"
-        rid = line_pool.search([("obj_id", "=", object_id), (field_src, "=", id)])
+        rid = line_pool.search([("obj_id", "=", object_id),
```

Comment:
Improve Variable name. 
Please use an informative variable.
Checked all and improve.